cmd/wallet-sdk-gomobile/openid4ci/interaction.go

```diff
@@ -50,23 +46,21 @@ func NewCredentialRequestOpts(userPIN string) *CredentialRequestOpts {
 // ActivityLogger is optional, but if provided then activities will be logged there.
 // If not provided, then no activities will be logged.
 type ClientConfig struct {
-	UserDID        string
 	ClientID       string
-	SignerCreator  api.DIDJWTSignerCreator
+	Crypto         api.Crypto
```

Comment:
The `api.Crypto` interface contains both `Sign` and `Verify` methods, but here it looks like we're only using the `Sign` method.  `api.Crypto` Should be broken down into `Signer` and `Verifier` interfaces so we don't require callers to implement unused methods.

Reply:
we need to do this in separate PR. and discuss this with @sudeshrshetty . Because it is against the initial design.

Reply:
I'm ok, we can split it.